test/gtest/ucp/test_ucp_tag_xfer.cc

```diff
@@ -123,6 +124,8 @@ class test_ucp_tag_xfer : public test_ucp_tag {
 
     void test_xfer_len_offset();
 
+    static const size_t BYTES_COUNT = 1148544;
```

Comment:
BYTES_COUNT -> MESSAGE_SIZE

Reply:
actually no need for this var anymore, since the test compare against a constant min threshold (50000)